tests/config/storage.php

```diff
@@ -2,26 +2,18 @@
 
 declare(strict_types=1);
 
-use Spiral\Storage\Server;
-
 return [
     'servers' => [
-        'local'     => [
-            'class'   => Server\LocalServer::class,
-            'options' => [
-                'home' => directory('public')
```

Comment:
Why didn't you use `directory('public').'uploads'`?

Reply:
> Why didn't you use `directory('public').'uploads'`?

Agree with you, fixed